main.py

```diff
@@ -66,6 +66,27 @@ def validate_google_user(provided_id_token):
 		return False
 	return True
 
+
+@app.route('/user/registration', methods=['POST'])
+def handle_registration():
+	if request.method == 'POST':
+		content = request.get_json()
+		# grabs values from JSON sent from mobile application
+		provided_gid = content.get("gid")
+		provided_name = content.get("displayName")
+		provided_avatar = content.get("avatar")
```

Comment:
Do we want to add some sort of validation or debugging statement for missing `content `fields?